client/src/plugins/vuetify.ts

```diff
@@ -10,12 +10,12 @@ export default new Vuetify({
     },
     themes: {
       light: {
-        primary: '#1669BB',
+        primary: '#1669bb', // same as $app-blue
         secondary: '#424242',
         accent: '#82B1FF',
-        error: '#FF5252',
+        error: '#d3272c', // same as $app-red
         info: '#2196F3',
-        success: '#4CAF50',
+        success: '#1a9031', // same as $app-green
```

Comment:
These Vuetify theme colours need to be kept in sync with our other theme colours. Please update as you go.